### PR TITLE
[FMS] [Camden] Cobrand Styling

### PR DESCRIPTION
**Pull request by lucascumsille**
This PR includes:

- Variables
- interactive components
- Mobile design
- Navigation bar and footer
- Emails

Here are some screenshots:

Screenshot 2022-10-25 at 09 00 14
Screenshot 2022-10-25 at 09 00 00
Screenshot 2022-10-25 at 08 59 44
Screenshot 2022-10-25 at 08 59 36
Screenshot 2022-10-25 at 08 59 12
Screenshot 2022-10-25 at 08 55 06
Screenshot 2022-10-25 at 08 54 52
Screenshot 2022-10-25 at 09 20 21
Screenshot 2022-10-25 at 09 20 04

Fixes: https://github.com/mysociety/societyworks/issues/3107

### CODE REVIEW

**Review by chrismytton — COMMENTED** [review]
[Feedback from Camden](https://3.basecamp.com/4020879/buckets/27695641/todos/5470577758#__recording_5483145245):

> ### Colours
>
> There are currently inconsistencies between small and large viewports:
> 
> * Background colours applied to search input panel for reporting a problem ([https://camden.staging.fixmystreet.com<https://camden.staging.fixmystreet.com/>](https://camden.staging.fixmystreet.com%3Chttps//camden.staging.fixmystreet.com/%3E)) on small viewports but not on large viewports
> * Background colours applied to dashboard at the bottom ([https://camden.staging.fixmystreet.com<https://camden.staging.fixmystreet.com/>](https://camden.staging.fixmystreet.com%3Chttps//camden.staging.fixmystreet.com/%3E)) on small viewports but not on large viewports
> Note: As we are in the middle of a redesign, we want to remove the previously suggested brand colour (teal) and replace it with a neutral colour scheme that matches the current site but would also not clash with the new design, once it's being introduced.
> The style guide now contains designs for the FMS home page that illustrate how the background colour should be applied across different viewports.
>
> ### Typography
>
> Font sizes/styles differ considerably in small and large viewports, but not in line with our style guide
> General rule: Breakpoint from small (S) to large (L) version is at 641 px viewport width. Viewports smaller than 641px use the small version of font and button styles. Viewports larger than 641px use the large version of font and button styles.
> 
> * `<h1>` (S) 32px | (L) 48px - semi-bold
> * `<h2>` (S) 24px | (L) 32px - semi-bold
> * `<h3>` (S) 18px | (L) 24px - semi-bold
> * `<h4>` (S) 16px | (L) 19px - semi-bold
> * `<p>`
> * large (can be used for introductory copy on pages) 18px | 24px
> * default (S) 16px | (L) 19px
> * small (S) 14px | (L) 16px
>
> ### Form input
>
> Focus style for form inputs as per style guide and example shown in page context. The currently implemented version is not accessible.
>
> ### Buttons
>
> Button size should change in line with typography changes at breakpoint 641px viewport size: button-height: (S) 45px | (L) 55px
> 'Report' button in header (small viewports only): Hover style is not accessible. Refer to header style in (updated) style guide and as shown in designed page example.
> 'Report a problem' and 'Local alerts' pages: Location button should use secondary button style, not primary button style.
> 'Report a problem': For 'Go' button next to search input see designed page on style guide.

**Comment by chrismytton** [comment]
> There are currently two different ‘Privacy and cookies’ links. The one in the footer goes to [camden.gov.uk](http://camden.gov.uk/)’s page, the one [here](https://camden.staging.fixmystreet.com/about/privacy) goes to a dedicated FMS page.
> The content of the second one appears to have very specific content related to FMS, whereas the first provides overall policy for Camden.
> Should there be two, or one that covers all points?

@lucascumsille Can we make sure that we're linking to Camden's privacy policy on their site please, thanks!

**Comment by chrismytton** [comment]
Oh and thanks for all your work on this so far @lucascumsille 😊

**Comment by lucascumsille** [comment]
Thanks for the feedback @chrismytton =)

Screenshot 2022-11-02 at 11 50 25
Screenshot 2022-11-02 at 11 50 11

According to the style provided by the client I have pushed some new changes and it should cover the points above.
Just in case I didn't change the "Go" button for an icon. Let me know you think we should. But so far this cobrand has taken more time that expected.

Here is the style provided by the client:
https://www.figma.com/file/If9ROhwOWsl2Y1p38S6Jsa/style-guide%3A-fix-my-street-v2?node-id=140%3A185

Let me know what you think =)

**Review by chrismytton — APPROVED** [review]

**Comment by lucascumsille** [comment]
Just in case @chrismytton I'm working on the new feedback. I'll send some commits tomorrow =)

**Comment by chrismytton** [comment]
@lucascumsille I've merged this branch into the `camden-reviewed` branch.

If you want to make further changes please create a new branch from the `camden-reviewed` branch and then create a pull request which has `camden-reviewed` as its base branch.

Thanks!

File: templates/web/camden/about/faq-en-gb.html [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+    <li>Urgent and emergency problems</li>
+    <li>Complaints about people, including anti-social behaviour</li>
+    <li>Issues with council services, such as bins and recycling</li>
+    <li>Proposals for change, eg to road layouts</li>
+    <li>Complaints about [% c.cobrand.council_name %]</li>
+</ul>
+
+<p><a name="emergencies"></a>For reporting urgent and emergency problems,
+such as a street light with the cover hanging off, or electrical wires where a bollard used to be,
+call 0117 922 2100, Monday to Friday, 8.30am to 6pm, or 0117 922 2050, all other times.
```
Comment (dracos) [inline templates/web/camden/about/faq-en-gb.html:31]: This is Bristol's FAQ. Unless there's a need for a specific one, the default fixmystreet-uk-council FAQ will be applied, so this file should not be necessary in an initial cobrand.

File: templates/web/camden/report/new/form_after_heading.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<p>Do not use this to <a href="/faq#emergencies">report emergencies outside of working hours</a>.</p>
```
Comment (dracos) [inline templates/web/camden/report/new/form_after_heading.html:1]: We don't want this template, we have an admin way of doing a reporting page emergency message for new cobrands.

File: templates/web/camden/footer_extra_js.html [hunk 3]
```diff
@@ -0,0 +1 @@
+[% PROCESS 'footer_extra_js_base.html' highways=1 cobrand_js=1 roadworks=1 ~%]
```
Comment (dracos) [inline templates/web/camden/footer_extra_js.html:1]: There is no cobrand JS file.

**Comment by dracos** [comment]
Sorry, just a few small comments as I now saw this, nothing major. Also there's a menu-black.svg - is that any different from the core one already present?

**Comment by lucascumsille** [comment]
> @lucascumsille I've merged this branch into the `camden-reviewed` branch.
> 
> If you want to make further changes please create a new branch from the `camden-reviewed` branch and then create a pull request which has `camden-reviewed` as its base branch.
> 
> Thanks!

Thanks @chrismytton  I just did that. I'll send the PR on Monday =)

**Comment by lucascumsille** [comment]
@dracos Thank Matthew, I got rid of the unnecessary files +  the logo icon =)

**Pull request merged**